ulm-semantics/test/execution.md

```diff
@@ -178,6 +179,23 @@ module ULM-TEST-EXECUTION
 
     rule (ulmCancel ~> expect_cancel) => .K
 
+    syntax Bytes ::= concat(BytesList)  [function, total]
+    rule concat(.BytesList) => b""
+    rule concat(B:Bytes + Bs:BytesList) => B +Bytes concat(Bs)
```

Comment:
This is a nitpick, so feel free take it or leave it 😄 

To me, `+` always stands for a verb. When you see it, it means that a function (or perhaps machine instruction) is being invoked --- whereas here, `+` is used as a data constructor, describing the shape of a value.

Given that feeling, my preference is for a generic infix operator, e.g. `|||` or `$` or `#`, etc.

Reply:
This may not look that good there, but it may make the tests look reasonable, IMO, see this example (I'm not sure if it was obvious where this list is being used):
https://github.com/Pi-Squared-Inc/rust-demo-semantics/pull/185/files#diff-bee3b842fdd29b8f22d8f3ab54cd51ab3d687b3a0a16e848ef877c5c4c20305fR18-R20

So, if you think that "$" would be better both here and in the tests, I'll change it.